website/package.json

```diff
@@ -13,31 +13,31 @@
   "author": "CloudQuery",
   "license": "MPL-2.0",
   "dependencies": {
-    "@heroicons/react": "1.0.6",
+    "@heroicons/react": "^2.0.14",
     "copy-to-clipboard": "3.3.3",
-    "isomorphic-unfetch": "3.1.0",
-    "next": "12.3.3",
-    "next-sitemap": "3.1.31",
-    "nextra": "2.0.0-beta.41",
-    "nextra-theme-docs": "2.0.0-beta.41",
+    "next": "^13.1.5",
+    "next-sitemap": "^3.1.47",
+    "nextra": "^2.2.13",
+    "nextra-theme-docs": "^2.2.13",
     "prism-react-renderer": "1.3.5",
     "prismjs": "1.29.0",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
     "react-hot-toast": "2.4.0"
   },
   "devDependencies": {
-    "@babel/core": "7.20.2",
-    "@types/node": "16.18.3",
-    "@types/react": "17.0.52",
+    "@babel/core": "7.20.12",
+    "@napi-rs/simple-git": "^0.1.8",
```

Comment:
`@napi-rs/simple-git` is added due to https://github.com/shuding/nextra/issues/1179#issuecomment-1405126693